src/components/dashboard/container/SourceSearch.jsx

```diff
@@ -9,7 +9,7 @@ import { clearSources } from '../../../redux/actions/sources/sourcesActionCreato
 import { fetchingOrganizations } from '../../../redux/actions/dictionaries/dictionaryActionCreators';
 import '../styles/index.css';
 
-import SideBar from '../components/SideNavigation';
+
```

Comment:
I think this is too much redundant space, if you don't mind you can remove one line for clean code

Reply:
however look good to me